lnbits/core/crud/settings.py

```diff
@@ -61,7 +61,21 @@ async def update_super_user(super_user: str) -> SuperSettings:
 
 
 async def delete_admin_settings(tag: Optional[str] = "core") -> None:
-    await db.execute("DELETE FROM settings WHERE tag = :tag", {"tag": tag})
+    await db.execute(
+        """
+        DELETE FROM system_settings
+        WHERE tag = :tag
+        AND (
+            tag != 'core'
+            OR id NOT IN (
```

Comment:
don't put logic in crud, just delete the settings and update the newly created settings with the old user